vispy/testing/_runners.py

```diff
@@ -111,6 +111,25 @@ def _unit(mode, extra_arg_string, coverage=False):
         os.rename(op.join(cwd, '.coverage'), out_name)
 
 
+def _docs():
+    """test docstring paramters
+    using vispy/utils/tests/test_docstring_parameters.py"""
+    print("warning: this imports vispy from the vispy installation. it is"
+          " recommended to setup vispy using 'python setup.py develop'"
+          " so that the latest sources are used automatically")
+    try:
+        # this should always be importable
+        from vispy.util.tests import test_docstring_parameters
+        print("Running docstring test...")
+        test_docstring_parameters.test_docstring_parameters()
+    except AssertionError as docstring_violations:
+        # the test harness expects runtime errors,
+        # not AssertionError. So wrap the AssertionError
+        # that is thrown by test_docstring_parameters()
+        # with a RuntimeError
+        raise RuntimeError(docstring_violations)
```

Comment:
The test runner does that already, doesn't it?
Here's my output on running it:

```
$ python make/make.py test docs
warning: this imports vispy from the vispy installation. it is recommended to setup vispy using 'python setup.py develop' so that the latest sources are used automatically
Running docstring test...
Passed

Testing succeeded (0 failed, 0 skipped) in 5.833 seconds
```